src/api/app/assets/stylesheets/webui/rouge.css.erb

```diff
@@ -0,0 +1 @@
+<%= Rouge::Themes::Github.render(scope: '.highlight') %>
```

Comment:
There should be a new line at the end of this file. This also applies to other files in this PR.

This is a short explanation on why it's needed: https://thoughtbot.com/blog/no-newline-at-end-of-file.

Reply:
Wow, that's good to know. Checking the files now.